### PR TITLE
Fix overlay header video playback

### LINKED ISSUE #454: Themes in 10.4 RC overlay video playback
**Describe the bug**

**To Reproduce**

1. Choose a theme on 10.4 RC (Haze is most noticeable however other themes like Dark also has transparency issues).
2. Play any video.

**Expected behavior**
See before picture.

**Logs**
N/A

**Screenshots**
Before:
image

After:
image 2

**System (please complete the following information):**
 - OS: Windows
 - Browser: Firefox/Vivaldi
 - Jellyfin Version: 10.4 RC

**Additional context**

### PR DESCRIPTION
**Pull request by grafixeyehero**
**Issues**
Fixes #454

### CODE REVIEW

**Comment by joshuaboniface** [comment]
@grafixeyehero The deminification caused conflicts with a previous PR. Can you remove the deminification from this PR and we'll do a full deminification soon?

**Comment by grafixeyehero** [comment]
@joshuaboniface as you suggested i update the PR

File: src/controllers/itemdetailpage.js [hunk 1]
```diff
@@ -1237,7 +1237,7 @@ define(["loading", "appRouter", "layoutManager", "connectionManager", "cardBuild
             view.addEventListener("viewbeforehide", function() {
                 events.off(apiClient, "message", onWebSocketMessage);
                 events.off(playbackManager, "playerchange", onPlayerChange);
-                libraryMenu.setTransparentMenu(false);
+                libraryMenu.setTransparentMenu(true);
```
Comment (JustAMan) [inline src/controllers/itemdetailpage.js]: How can this be a fix? According to `git blame` this `setTransparentMenu(false)` was so from the get-go, how can it be broken in 10.4-RC?

Comment (grafixeyehero) [reply src/controllers/itemdetailpage.js]: i think this was broken form the start.

Comment (JustAMan) [reply src/controllers/itemdetailpage.js]: If I read the bug correctly, it says that menu started being opaque after 10.3.7... How can it be broken from the beginning?..

Comment (grafixeyehero) [reply src/controllers/itemdetailpage.js]: i will check it on 10.3.7

Comment (grafixeyehero) [reply src/controllers/itemdetailpage.js]: yeah you are correct. This was removed https://github.com/jellyfin/jellyfin-web/pull/325/files

**Comment by thornbill** [comment]
Proposed fix using css: https://github.com/jellyfin/jellyfin-web/pull/478

**Comment by anthonylavado** [comment]
Since @thornbill has added #478, do we still want this PR? Or can both exist together?

**Pull request closed without merging**